tensorflow-mixin/config.libsonnet

```diff
@@ -1,5 +1,7 @@
 {
   _config+:: {
+    enableMultiCluster: false,
+    tensorflowSelector: if self.enableMultiCluster then 'job=~"$job", cluster=~"$cluster"' else 'job=~"$job"',
```

Comment:
I believe we're missing the instance matcher if not using multi cluster mode, e.g. all the replacements in the dashboards replace both job and instance with only job.

Is this intended?

Reply:
It's not intended, I think the `instance` label is accounted for within this `getMatcher` function, but I might be mistaken, should we be including `instance=~"$instance"` within the selectors in the `config.libsonnet` file instead? 
```
local getMatcher(cfg) = '%(tensorflowSelector)s, instance=~"$instance"' % cfg;
```

Reply:
Great point, you're totally right! Sorry Austin that's on me 😅 

I'm blaming it on PTO-brain